src/data/species.yaml

```diff
@@ -53,9 +53,30 @@
     0a591d04-1904-466d-a198-e172a11a424e:23,
     dfca4764-06e7-4ee3-9ac8-aec65723bdec:100
   generatedId: fd6b29ee-aaf8-4a23-b8bd-990797ef915f
+- name: Dewback
+  player: No
+  index: Stay on Target:85
```

Comment:
Can you please change the index name to be the `generatedId` of the book? We made this change awhile ago to make clickable books in the index columns easier.